modules/twinklewarn.js

```diff
@@ -1144,20 +1149,21 @@ Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCatego
 		// no categories, just create the options right away
 		createEntries(Twinkle.warn.messages[value], sub_group, true);
 	} else if (value === 'singlecombined') {
-		createEntries(Twinkle.warn.messages['singlenotice'], sub_group, true);
-		createEntries(Twinkle.warn.messages['singlewarn'], sub_group, true);
+		var singletMessages = $.extend({}, Twinkle.warn.messages['singlenotice'], Twinkle.warn.messages['singlewarn']);
```

Comment:
should probably be written as `Twinkle.warn.messages.singlenotice`. This bracket notation is not used anywhere in TW except where it's necessary.

Reply:
Dealt with at https://github.com/azatoth/twinkle/pull/759/files#r352365578